travis-ci

```diff
@@ -0,0 +1 @@
+ci
```

Comment:
is this a symlink?

Reply:
ok, missed your mention of it in the end, thank you!

Reply:
I guess symlink will be required until travis-ci is removed in https://github.com/libbpf/ci/search?q=travis-ci

Reply:
> I guess symlink will be required until travis-ci is removed in https://github.com/libbpf/ci/search?q=travis-ci

Yep, that is correct.